### ISSUE TITLE
Total amount services - Delete service

### ISSUE THREAD

**Issue by RomaneJAUD** [Bulbthings/bulbthings-locoto#87]
Hi @matthewjameslockwood 
When we delete a service (for example Véhicule de remplacement), the amounts of other services are modified (for example : Perte financière) 

In this example : 
Perte financière = 8,20€
I delete Véhicule de remplacement and now Perte financière = 6,90€

Could you change please ? Every services will be independent please

**Comment by matthewjameslockwood** [comment]
Hi @RomaneJAUD, I've taken a look at this and I can't seem to find any issues with it 😄, is there any chance you have a contract reference number where I can take a further look?

**Comment by RomaneJAUD** [comment]
Hi @matthewjameslockwood 
Sorry I haven't no longer the case

**Comment by RomaneJAUD** [comment]
Sorry @matthewjameslockwood i just found an example.
It's on Loc'oto environnement (not Loc'oto TEST): NTE 1007

1. ![Screenshot](https://user-images.githubusercontent.com/87650448/131982814-b0a5a0b0-6292-4d5a-8da4-12adf2a485e2.png)
2. We delete assurance and "Entretien" total amount = 51,88€ instead of 49,34€

**Comment by matthewjameslockwood** [comment]
Hi @RomaneJAUD, have these services/the above contract been created from an import?

**Comment by RomaneJAUD** [comment]
Yes, but with or without import, the amounts should no longer be able to be reclaculated if is a contract (not quotation)

**Comment by matthewjameslockwood** [comment]
Ah, okay. have the maintenance fluidity grids changed at all? @RomaneJAUD, I've taken a deeper look into this and I can assume that that could the issue.

**Comment by matthewjameslockwood** [comment]
It all originates from this sheet:
https://docs.google.com/spreadsheets/d/1k5QzR13PQbux2xBWB5KlgneHTI2D79c7tsWS3a5McJ0/edit
I've requested access in the meantime so I can investigate further.

**Comment by RomaneJAUD** [comment]
Maintenance fluidity grids haven't changed but we may have amounts that don't match the grids.
For contract (not quotation) whether or not the grids change, the amounts must remain fixed

**Comment by RomaneJAUD** [comment]
Ok I opened the access. But I think it's necessary to block the calcul when quotation has been signed (even if grid changes). We have to keep validate total amount

**Comment by matthewjameslockwood** [comment]
Okay, just to be clear the "Entretien" amount uses the following formula:

entretien total gross amount = vehicle purchase price * grid percentage amount

The total is changing either because either values have changed. I will fix the maintenance grid if the quotation has been signed. What status change should that be? The status after 'Quote'?

**Comment by RomaneJAUD** [comment]
Yes : vehicle catalogue purchase price * grid percentage _(depending on the km / duration)_

- If it's a quotation : total is changing either because either values have changed.

- If the status is after 'Quote' _(Devis validé, Commande envoyée (concession), Commande confirmée (concession), Commande confirmée (client), En production, En attente PV des mines, Véhicule immatriculé, Rendez-vous à planifier, En cours de livraison, À la route, Véhicule à chiffrer,Restitué, Contrat ajusté, Vendu, Épave, Véhicule volé)_ : amounts cannot change
